R/estimateParams.R

```diff
@@ -0,0 +1,55 @@
+#' Preparation of a Data Set to Compute Log-likelihood
+#'
+#' @param data (`data.frame`)\cr data frame containing entry and exit times of an illness-death model.
```

Comment:
```suggestion
#' @param data (`data.frame`)\cr containing entry and exit times of an illness-death model.
```
(duplicate with type spec)